src/test/java/GitHubTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+public class GitHubTest extends BaseTest {
+
+    @Test
+    public void gitHubTest() throws InterruptedException {
+        HomePage homePage = new HomePage(driver);
+        homePage.goToLoginPage();
+
+        LoginPage loginPage = new LoginPage(driver);
+        Assertions.assertTrue(loginPage.getLogo().isDisplayed());
+        loginPage.successfulLogin("HW10Test", "Nthvbyfnjh219");
+
+        MainPage mainPage = new MainPage(driver);
+        mainPage.goToRepositoriesPage();
+
+        RepositoriesPage repositoriesPage = new RepositoriesPage(driver);
+        repositoriesPage.goToRepositoriesPage();
+
+        IssuesPage issuesPage = new IssuesPage(driver);
+        issuesPage.newIssueBtn();
```

Comment:
method newIssueBtn() should be name as action - clickOnIssueButton() and for others the same